doc/overview/qfield_overview.rst

```diff
@@ -0,0 +1,80 @@
+:Author: OSGeoLive
```

Comment:
You can add your self as author that meand Matthias Kuhn

Reply:
```suggestion
:Author: Matthias Kuhn
```